lib/puppet/face/parser.rb

```diff
@@ -113,7 +113,7 @@
         if missing_files.empty?
           dumps
         else
-          dumps + "One or more file(s) specified did not exist:\n" + missing_files.collect { |f| "   #{f}" }.join("\n")
+          dumps + _("One or more file(s) specified did not exist:\n") + missing_files.collect { |f| "   #{f}" }.join("\n")
```

Comment:
I expect translators won't mess with embedded newlines - does that seem right to you?